lib/drizzle/application.ex

```diff
@@ -36,4 +32,37 @@ defmodule Drizzle.Application do
       {Drizzle.TodaysEvents, []}
     ]
   end
+
+  def wait_for_internet(iface) do
+    IO.puts "waiting for #{iface} to become ready..."
+    VintageNet.subscribe(["interface", iface])
+    receive do
+      {VintageNet, ["interface", ifname, "connection"], _oldstate, :internet, _} ->
+        IO.puts "===> #{ifname} configured <==="
+        start_phoenix()
+      _ -> wait_for_internet(iface)
+    end
+  end
+
+  defp prepare_network do
+    IO.puts "===> prepare_network <==="
+    if "wlan0" in VintageNet.all_interfaces() do
+        # start the WiFi wizard if the wireless interface is not configured
+        if "wlan0" in VintageNet.configured_interfaces() do
```

Comment:
```suggestion
        if "wlan0" not in VintageNet.configured_interfaces() do
```
Logic is a little off here. With this, the wizard will come on when wlan is already configured

Reply:
right, spot on :) will fix

Reply:
fixed